dozer-ingestion/src/connectors/postgres/schema_helper.rs

```diff
@@ -94,9 +94,9 @@ impl SchemaHelper {
                 let table_name: String = row.get(0);
                 let column_name: String = row.get(1);
 
-                tables_columns_map
-                    .get(&table_name)
-                    .map_or(true, |table_info| table_info.contains(&column_name))
+                tables_columns_map.get(&table_name).map_or(true, |columns| {
```

Comment:
@karolisg Error was showing something else about `Replication Type` ? 

Should we add a unit test case to avoid regression in initialization ?

Reply:
Schema helper was expecting `None` or `not empty vec`. As you passed empty array, no columns were fetched and it was not possible determine replication type.